### PR TITLE
added a changelog for the pack readme validation

### PR DESCRIPTION
**Pull request by omerKarkKatz**
## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related PR
fixes: https://github.com/demisto/demisto-sdk/pull/1887

## Description
added a changelog for the pack readme - [dev - bug] #1887 PR

## Must have
- [ ] Tests
- [ ] Documentation
- [ ] [VS Code Extension](https://docs.gitlab.com/ee/ci/yaml/#tags)
  - [ ] Functionality implemented in the extension - <u>\<link-to-pr-here\></u>
  - [ ] N/A - Functionality cannot be implemented in the extension because <u>\<add-reason-here\></u>

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -7,6 +7,7 @@
 * Added a validation to **validate**, checking that each script, integration and playbook have a README file. This validation only runs when the command is called with either the `-i` or the `-g` flag.
 * Fixed a regression issue with **doc-review**, where  the `-g` flag did not work
 * Improved the detection of errors in **doc-review** command.
+* When **validate** is validating *pack unique files* it will check for README existence only for packs that contain playbooks or writen by partners.
```
Comment (dorschw) [inline CHANGELOG.md:10]:
```suggestion
* Fixed an issue in **validate**, where README existence was only checked in packs that contain playbooks, or written by partners. 
```

Comment (omerKarkKatz) [reply CHANGELOG.md:10]: That is not really what i have done. the validation was once triggered for all packs containing use cases or written by partners. 
now it validates only for packs how contain playbooks or were written by partners. 
also made a better more clear error message. 
@dorschw

Comment (dorschw) [reply CHANGELOG.md:10]: got it. plesae push the updated message.

File: CHANGELOG.md [hunk 2]
```diff
@@ -7,6 +7,7 @@
 * Added a validation to **validate**, checking that each script, integration and playbook have a README file. This validation only runs when the command is called with either the `-i` or the `-g` flag.
 * Fixed a regression issue with **doc-review**, where  the `-g` flag did not work
 * Improved the detection of errors in **doc-review** command.
+* Fixed an issue in **validate**, where README is not empty, is only checked in packs that contain playbooks, or written by partners.
```
Comment (dorschw) [inline CHANGELOG.md:10]:
```suggestion
* Fixed an issue in **validate**, where non-empty README files were only checked in packs that contain playbooks, or written by partners. 
```

Comment (omerKarkKatz) [reply CHANGELOG.md]: @dorschw don't think this is the best wording. please look again and confirm if ok by you.

**Review by dorschw — APPROVED** [review]

**Pull request merged**